invoicer_project/user/views.py

```diff
@@ -65,6 +68,57 @@ def create(self, request):
             refresh = RefreshToken.for_user(user)
             return JsonResponse({'refresh': str(refresh), 'access': str(refresh.access_token)}, status=200)
         return JsonResponse({"message": "Invalid credentials"}, status=400)
+    
+    @action(detail=True, methods=['post'])
+    def send_confirmation_code(self, request):
+        data = JSONParser().parse(request)
+        email = data.get('email')
+        try:
+            user = User.objects.get(email=email)
+        except User.DoesNotExist:
+            user = None
+        if user:
+            code = ''
```

Comment:
`code` is always empty string! Is it correct?

Reply:
Done